server/rules_test.go

```diff
@@ -19,15 +19,14 @@ func TestParseOneRule(t *testing.T) {
 	routeResult, err := rules.Parse(expression)
 
 	if err != nil {
-		t.Fatal("Error while building route for Host:foo.bar")
+		t.Fatal("Error while building route for Host:foo.bar", err)
```

Comment:
For consistency reasons: this PR already changes a few

`t.Error("something" + err)`

into

`t.Errorf("something: %s", err)`

so it'd only be consequential to stick to the latter format here too.